00-RELEASENOTES

```diff
@@ -11,6 +11,25 @@ CRITICAL: There is a critical bug affecting MOST USERS. Upgrade ASAP.
 SECURITY: There are security fixes in the release.
 --------------------------------------------------------------------------------
 
+================================================================================
+Redis 6.0.17 Released Mon Jan 16 12:00:00 IDT 2023
```

Comment:
```suggestion
Redis 6.0.17 Released Mon Jan 16 12:00:00 IST 2023
```

Reply:
the time is inaccurate anyway, so i don't really care about off by one in the timezone.